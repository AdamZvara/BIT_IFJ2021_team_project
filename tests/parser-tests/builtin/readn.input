require "ifj21"

function main()
	local a : number = readn()
	while a < 20.2 do
		a = a * a
		a = a / 2.2
		write(a,"\n")
	end
	a = readn()
	write(a,"\n")
end


main()
