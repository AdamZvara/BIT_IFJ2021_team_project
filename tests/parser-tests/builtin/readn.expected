0x1.3ccccccccccccp+2
0x1.6466666666664p+3
0x1.c311999999993p+5
0x1.4p+1
